.circleci/ci-tests/terraform-generated-aws/module-docdb.yaml

```diff
@@ -0,0 +1,9 @@
+environments:
+  - name: env-tg
+    path: "environment.yaml"
+    variables:
+      max_containers: 2
+name: module-docdb
+modules:
+  - name: tgdocdb
+    type: aws-documentdb
```

Comment:
if the module doesn't have any mandatory attribute, then this file could be generated automatically by circleci.
this will also make adding new test case easier for such modules.

Reply:
This is something which can be looked up on later, if and when modules are added more frequently.
As a basic test, I have only added the modules which are used much more frequently.

Reply:
ok, it can be added later, but we need to test all the modules with `tf plan` as part of this effort.
the goal is to find the issues and either mark these modules with error as unsupported for export and then switch the command from beta to available.